CMakeLists.txt

```diff
@@ -110,7 +132,7 @@ file(GLOB ${PROJECT_NAME}_HEADERS
     header/*.h
 )
 
-if (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang") #------------------------------------- Clang
+if (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang"  AND UNIX AND NOT APPLE ) #------------------------------------- Clang
     find_library(CLANG_LIBCPP
```

Comment:
Instead of specifying dependencies on `REQ_LLVM_LIBRARIES` in each consuming CMakeLists, better to add PUBLIC linkage to `REQ_LLVM_LIBRARIES` in `actor-zeta` target, like
```cmake
target_link_libraries(${PROJECT_NAME}
PUBLIC REQ_LLVM_LIBRARIES
```